SLHCUpgradeSimulations/Configuration/python/fixMissingUpgradeGTPayloads.py

```diff
@@ -23,14 +23,14 @@ def fixDTAlignmentConditions(process):
     process.GlobalTag.toGet.extend( cms.VPSet(
             cms.PSet(record = cms.string("DTAlignmentErrorExtendedRcd"),
                      tag = cms.string("MuonDTAPEObjectsExtended_v0_mc"),
-                     connect = cms.string("frontier://FrontierProd/CMS_COND_ALIGN_000")
+                     connect = cms.string("frontier://FrontierProd/CMS_CONDITIONS")
                  )
             )
     ),
     process.GlobalTag.toGet.extend( cms.VPSet(
             cms.PSet(record = cms.string("DTRecoUncertaintiesRcd"),
                      tag = cms.string("DTRecoUncertainties_True_v0"),
-                     connect = cms.string("frontier://FrontierProd/CMS_COND_DT_000")
+                     connect = cms.string("frontier://FrontierProd/CMS_CONDITIONS")
```

Comment:
this could be dropped: a tag containing the same payload is already in use in all MC GT:

````
80X_mcRun1_design_Queue	Queue	DTRecoUncertaintiesRcd	-	DTRecoUncertainties_v1_mc
80X_mcRun1_HeavyIon_Queue	Queue	DTRecoUncertaintiesRcd	-	DTRecoUncertainties_v1_mc
80X_mcRun1_pA_Queue	Queue	DTRecoUncertaintiesRcd	-	DTRecoUncertainties_v1_mc
80X_mcRun1_realistic_Queue	Queue	DTRecoUncertaintiesRcd	-	DTRecoUncertainties_v1_mc
80X_mcRun2_asymptotic_Queue	Queue	DTRecoUncertaintiesRcd	-	DTRecoUncertainties_v1_mc
80X_mcRun2_design_Queue	Queue	DTRecoUncertaintiesRcd	-	DTRecoUncertainties_v1_mc
80X_mcRun2_HeavyIon_Queue	Queue	DTRecoUncertaintiesRcd	-	DTRecoUncertainties_v1_mc
80X_mcRun2_startup_Queue	Queue	DTRecoUncertaintiesRcd	-	DTRecoUncertainties_v1_mc
76X_upgrade2017_design_Queue	Queue	DTRecoUncertaintiesRcd	-	DTRecoUncertainties_v1_mc
````